src/ansys/geometry/core/designer/body.py

```diff
@@ -1208,6 +1208,9 @@ def wrapper(self: "Body", *args, **kwargs):
 
         return wrapper
 
+    def _reset_tessellation_cache(self):  # noqa: N805
+        self._template._tessellation = None
+
```

Comment:
Why is this method not used?

Reply:
@RobPasMue there is a method with the same name that is used as a decorator in body.py, so this is just a method to be used outside that module